www/templates/www/thanks.html

```diff
@@ -1,7 +1,11 @@
 {% extends "www/base.html" %}
 {% load i18n %}
 {% block content %}
-<p>
-    {% trans 'Registration successful! Please check your e-mail (and spam folder) for instuctions.' %}
-</p>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-lg-6 offset-lg-3">
+        <h1>{% trans 'Registration successful! Please check your e-mail (and spam folder) for instuctions.' %}</h1>
```

Comment:
Should be application, not registration.. My bad, I think I wrote the string.

Reply:
https://github.com/TampereHacklab/mulysa/pull/103/commits/0367f169fa5e0d710d22c86704ee939c98dc0bd8